src/platform/bouffalolab/BL602/NetworkCommissioningDriver.cpp

```diff
@@ -157,12 +165,18 @@ CHIP_ERROR BLWiFiDriver::ConnectWiFiNetwork(const char * ssid, uint8_t ssidLen,
 
     char wifi_ssid[64] = { 0 };
     char passwd[64]    = { 0 };
+
     // Set the wifi configuration
+    wifi_mgmr_sta_disconnect();
+    vTaskDelay(500);
```

Comment:
500 ticks here is a bit unclear  since it depends on portTICK_PERIOD_MS.
Could you add an explicit in-ms delay instead of something that varies based on OS tick rate?

Why is 500 a correct value to use?

Reply:
How was this resolved?

Reply:
Was looking at an old version - this was indeed updated.